tests/test_event_track.py

```diff
@@ -0,0 +1,31 @@
+"""Copyright 2015 Rafal Kowalski"""
+import datetime
+import unittest
+
+from setup_database import Setup
+
+from open_event import app
+from open_event.models import db
+from open_event.models.track import Track
+from open_event.models.event import Event
+
+
+class OpenEventTestCase(unittest.TestCase):
+    def setUp(self):
+        self.app = Setup.create_app()
+
+    def tearDown(self):
+        Setup.drop_db()
+
+    def test_api_tracks(self):
+        ev = Event(name="event1", start_time=datetime.datetime(2003, 8, 4, 12, 30, 45), end_time=datetime.datetime(2003, 8, 4, 12, 30, 45))
+        tr = Track(name="name", event_id=1, description="description")
+        with app.app_context():
```

Comment:
This should be wrapped in a try/except block and handle those errors if they arise. For instance, if adding the event fails, you can give a different test failure message than if a track fails